code/game/gamemodes/blob/powers.dm

```diff
@@ -8,7 +8,7 @@
 // Power verbs
 
 /mob/camera/blob/proc/place_blob_core(var/point_rate = base_point_rate, var/override = 0)
-	if(placed)
+	if(src.placed && override != -1) //byond is a pile of dicks so I need to do unimplied src for some reason
```

Comment:
somebody please tell me why the src was needed here because if you proccalled it or used the admin make blob verb it would runtime unless it was src.placed

I'm honestly really curious as to why